services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServer.java

```diff
@@ -153,7 +156,9 @@ public void testGetTablesTypes() throws Exception {
       Assert.assertTrue(stream.next());
       VectorSchemaRoot root = stream.getRoot();
 
-      final ImmutableList<TableType> tableTypes = ImmutableList.of(TableType.TABLE, TableType.SYSTEM_TABLE, TableType.VIEW);
+      final List<TableType> tableTypes = Arrays.stream(TableType.values())
```

Comment:
Can we use the hardcoded version for the tests, since then we're validating the known items. If someone were to add another undesired element (perhaps ERROR), we don't want that to come back.

Reply:
Fixed